pallets/parachain-staking/src/tests.rs

```diff
@@ -4070,7 +3978,6 @@ fn parachain_bond_inflation_reserve_matches_config() {
 				},
 			];
 			assert_eq_events!(expected.clone());
-			assert_eq!(Balances::free_balance(&11), 1);
```

Comment:
Removing this because account 11 doesn't have any special role to play anymore. It used to be the account that was holding the parachain bond.